.github/workflows/daily-testing.yml

```diff
@@ -0,0 +1,64 @@
+# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: daily-testing
+
+on:
+  push:
+    branches:
+      - gha-testing
+  schedule:
+    # daily testing to occur at 7:30 CST every day
+    - cron:  '30 13 * * *'
```

Comment:
Could we do once-a-week instead? Daily could be a little extreme

Reply:
Done! Changed it to test every Monday.